code/browser.py

```diff
@@ -18,6 +18,28 @@ def is_url(url):
         return False
 
 
+@ctx.action_class("user")
+class UserActions:
+    def browser_address_fallback() -> str:
+        try:
+            address = actions.browser.address()
+            if address:
+                return address
+        except Exception:
+            pass
+        actions.browser.focus_address()
+        actions.sleep("50ms")
+        with clip.capture() as s:
+            actions.edit.copy()
+        actions.key("escape")
+        actions.key("tab")
```

Comment:
What do you expect this to do? Are you sure it will always put the focus where you want it even across browsers and platforms?

Reply:
You are right, this is a bad idea. It doesn't even return the focus to the document, at least in Firefox, which is what I would need.